scripts/networks/deploymentFunctions.ts

```diff
@@ -1833,3 +1835,45 @@ export const prechecks: IDeploymentFunction = async function (params: IDeploymen
   }
   return {}
 }
+
+export const extractTokenConfig = async function (params: IDeploymentParams): Promise<ITokenConfig[]> {
+  const getContract = await getContractFromSection(params.existing)
+  const LR_old = await getContract<Types.LiquidityReceiverV1>(Sections.LiquidityReceiverOld, "LiquidityReceiverV1");
+  const LR_new = await getContract<Types.LiquidityReceiver>(Sections.LiquidityReceiverNew, "LiquidityReceiver");
+  let tokenConfigs = [] as ITokenConfig[]
+
+  const pyroTokenFactory = await ethers.getContractFactory("PyroToken")
+  const behodlerTokenSection = params.existing[sectionName(Sections.BehodlerTokens)]
+  const behodlerTokenNames = Object.keys(behodlerTokenSection)
+  for (let i = 0; i < behodlerTokenNames.length; i++) {
+    const tokenName = behodlerTokenNames[i] as contractNames
+    const token = await getContract<Types.ERC20>(Sections.BehodlerTokens, tokenName, "ERC20")
+
+    let displayName: string = tokenName
+    let pyroDisplayName = 'pyro(' + tokenName + ')'
```

Comment:
Any particular reason you prefer using string concatenation over template literals? I find `pyro(${tokenName})` to be much more readable than `'pyro(' + tokenName + ')'`, but maybe that's just me.